### PR TITLE
Reorganize and improve documentation

### PR DESCRIPTION
**Pull request by jonas**

### CODE REVIEW

File: docs/src/paradox/obtaining-bindgen/docker-container.md [hunk 1]
```diff
@@ -1,32 +0,0 @@
-# Using Docker Container
```
Comment (kornilova-l) [inline docs/src/paradox/obtaining-bindgen/docker-container.md]: Will we remove support for bindgen images? (I mean pushing new versions to scalabindgen docker repo)

File: docs/src/paradox/getting-started.md [hunk 2]
```diff
@@ -1,18 +1,25 @@
-# Command Line Usage
+# Getting Started
+
+No setup is required in order to use to use `scala-native-bindgen`. Statically linked
+executables are provided with each release for Linux and macOS. Head over to the [releases page] to download the latest version for your platform.
+
+ [releases page]: https://github.com/kornilova-l/scala-native-bindgen/releases
+
+## Command Line Usage
```
Comment (kornilova-l) [inline docs/src/paradox/getting-started.md]: Probably we should mention here that there are two ways to obtain bindgen and provide a link to `using-generated-bindings` page (I will update this page soon)

File: docs/src/paradox/obtaining-bindgen/docker-container.md (hunk 1 above)
Comment (jonas) [reply docs/src/paradox/obtaining-bindgen/docker-container.md]: The docker image is less useful now that we publish statically linked executables.

Comment (jonas) [reply docs/src/paradox/obtaining-bindgen/docker-container.md]: ... so yeah, we could remove support for the docker images.

File: docs/src/paradox/getting-started.md (hunk 2 above)
Comment (jonas) [reply docs/src/paradox/getting-started.md]: Yes, working on that.

Comment (jonas) [reply docs/src/paradox/getting-started.md]: All the code snippets on the `using-generated-bindings` page are now tested and/or generated.

**Comment by jonas** [comment]
I'll go ahead and merge this so I can do an RC.

**Pull request merged**